README.md

```diff
@@ -1,11 +1,12 @@
 python-for-android
 ==================
-
 [![Build Status](https://travis-ci.org/kivy/python-for-android.svg?branch=develop)](https://travis-ci.org/kivy/python-for-android)
 [![Coverage Status](https://coveralls.io/repos/github/kivy/python-for-android/badge.svg?branch=develop&kill_cache=1)](https://coveralls.io/github/kivy/python-for-android?branch=develop)
 [![Backers on Open Collective](https://opencollective.com/kivy/backers/badge.svg)](#backers)
 [![Sponsors on Open Collective](https://opencollective.com/kivy/sponsors/badge.svg)](#sponsors)
 
+*Added Bcrypt Recipe*
```

Comment:
Why are these change in the README.md? Could you revert all changes of it if not needed.

Reply:
@AndreMiras  I removed the redundant line.

Reply:
Thanks, there's still one change in the README.md to revert. Your diff removes one line when it should simply not touch the README.md at all as it brings noise we don't want in the history. Could you fix that one too before we merge?

Reply:
@AndreMiras Done:)

Reply:
Thanks :beers: